worker/globalclockupdater/manifold.go

```diff
@@ -68,8 +64,8 @@ func (config ManifoldConfig) start(context dependency.Context) (worker.Worker, e
 		return nil, errors.Trace(err)
 	}
 
-	// We don't need anything from raft directly, but ensure it's
-	// running before continuing.
+	// This enforces a dependency on the Raft forwarder,
+	// effectively ensuring this worker is only active on the Raft leader.
```

Comment:
It also ensures that Raft is running properly before we start updating the clock, which I do think is important.

Reply:
Included that snippet.